src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -122,7 +123,8 @@ internal static Configuration LoadConfiguration(string basePath, string json)
                 ReadOption(root, Configuration.Options.SkipCSProjPackageReference),
                 ReadOption(root, Configuration.Options.LowLevelClient),
                 ReadOption(root, Configuration.Options.SingleTopLevelClient),
-                MgmtConfiguration.LoadConfiguration(root)
+                MgmtConfiguration.LoadConfiguration(root),
```

Comment:
Let's keep Mgmt configuration at the end.

Reply:
Updated.